tasks/system/general/set_limits.yml

```diff
@@ -5,6 +5,7 @@
     limit_type: "{{ item.limit_type }}"
     limit_item: "{{ item.limit_item }}"
     value: "{{ item.value }}"
+    dest: "/etc/security/limits.d/70-cloudenterprise.conf"
```

Comment:
I gues you added this here for a better seperation, maybe you can explain your thought on that a bit?

Reply:
Apologies for answering late!

I actually intended to only submit the PR for firewalld ... honestly don't know how the other modifications got mixed in, sorry!
You're absolutely correct that these should be separate issues.

Reply:
Simply close the PR and i'll give it a try again